### PR TITLE
[ADD] Option to treat weight and bias of a layer jointly in KFAC

### LINKED ISSUE #53: Support treating weights and biases of a layer jointly in KFAC

### PR DESCRIPTION
**Pull request by f-dangel**
Eventually, this will resolve #53.

At the moment, I am struggling to get the tests running and would benefit from a second pair of eyes to look through the code. At the moment, the tests seem to execute for both `separate_weight_and_bias = [False, True]`, but the matrix produced by KFAC is wrong for `True`.

I believe this is related to the order of parameters. But I think I am actually arranging them in the correct order 😂, so maybe I'm barking up the wrong tree. In any case, it would be great if you could skim the code and look for anything obvious I missed.

### CODE REVIEW

**Comment by f-dangel** [comment]
**Update:** I found the problem and fixed everything. Ready for review.

File: test/test_kfac.py [hunk 1]
```diff
@@ -9,11 +10,11 @@
 from torch.nn import Module, MSELoss, Parameter
 
 from curvlinops.examples.utils import report_nonclose
-from curvlinops.ggn import GGNLinearOperator
 from curvlinops.gradient_moments import EFLinearOperator
 from curvlinops.kfac import KFACLinearOperator
 
 
+@mark.parametrize("separate_weight_and_bias", [True, False], ids=["separate", "joint"])
```
Comment (runame) [inline test/test_kfac.py:17]: The `ids` could a bit more verbose here, e.g. `ids=["separate_bias", "joint_bias"]`.

**Review by runame — APPROVED** [review]
Sorry was asleep :D But LGTM!

File: test/test_kfac.py (hunk 1 above)
Comment (f-dangel) [reply test/test_kfac.py:17]: Incorporated.

**Pull request merged**